### PR TITLE
Create Noahk2024

### PR DESCRIPTION
**Pull request by Noahk2024**
## Description
I'm building an automatic plant care growth system that automates all plant care using a raspberry pi and 3d printed parts for the body.

## Checklist before submitting proposal:

- [x] I am a current high school student (or younger).
- [x] I have submitted an RSVP at <https://hackclub.com/winter>
- [x] I am a member of the Hack Club Slack (Join at <https://hackclub.com/slack>)
- [x] I have self-reviewed my project proposal.

### CODE REVIEW

File: Noahk2024 [hunk 1]
```diff
@@ -0,0 +1,66 @@
+---
+name: "@Noahk2024"
+project: "Automated Plant Growth System"
+---
+
+# Automated Plant Growth System
+
+## Summary
+I am going to build a automated plant growth pod that automates plant care for indoor gardening. It has a water resevoir in the bottom of the pod with the planting soil
```
Comment (bellesea) [inline Noahk2024:9]:
```suggestion
I am going to build an automated plant growth pod that automates plant care for indoor gardening. It has a water reservoir in the bottom of the pod with the planting soil 
```

**Review by bellesea — CHANGES_REQUESTED** [review]
hiii @Noahk2024 woah i love this idea!

could you please:
- make this a markdown file by adding .md to the filename?
- make your budget a [markdown table](https://www.markdownguide.org/extended-syntax/#tables)?

also, how do you plan on controlling the plant growth LEDs? and, are you referencing any guides? plus is your tube long enough? and, would your 3d printer be big enough to print the reservoir?

thanks!

**Comment by Noahk2024** [comment]
> hiii @Noahk2024 woah i love this idea!
> 
> could you please:
> 
> * make this a markdown file by adding .md to the filename?
> * make your budget a [markdown table](https://www.markdownguide.org/extended-syntax/#tables)?
> 
> also, how do you plan on controlling the plant growth LEDs? and, are you referencing any guides? plus is your tube long enough? and, would your 3d printer be big enough to print the reservoir?
> 
> thanks!

**Comment by Noahk2024** [comment]
Sure thing. The plant growth LEDs come pre-installed with a timer that you can enable, and alternatively since the LED will be plugged into the raspberry pi I can toggle power with that port. All the adafruit components come with documentation and I will be doing this mostly from scratch besides repurposed sample code. The tubing should be long enough, and I will be printing parts in pieces to accommodate the size

**Comment by bellesea** [comment]
> Sure thing. The plant growth LEDs come pre-installed with a timer that you can enable, and alternatively since the LED will be plugged into the raspberry pi via USB I can toggle power with that port. All the adafruit components come with documentation and I will be doing this mostly from scratch besides repurposed sample code. The tubing should be plenty long enough, and I will be printing parts in segments to accommodate the size of the printer, attaching them together using superglue or an epoxy adhesive. As for the crevices created by this, I will cover them with caulking to further seal the reservoir and leak-proof it.

hii @Noahk2024 thank you so much for the rapid response! on the LEDs, i'm not super sure if that would work? https://forum.arduino.cc/t/arduino-controlling-the-power-supply-of-a-usb-light/956244/5 i haven't personally done this so i'm not sure but would you be able to look a little more into it to make sure it would work or find an alternative?

could you please commit whatever you find into the files? (so if you're sure it'll work add a note under ## plan why it will and if you choose an alternative please update too)

thank you!

**Comment by Noahk2024** [comment]
I'm positive it would work because I can use the uhubctl utility (https://github.com/mvp/uhubctl) to control power on my USB ports. https://funprojects.blog/2021/04/26/control-usb-powered-devices/

**Comment by bellesea** [comment]
> 

great, thank you so much!

**Pull request merged**